packages/airnode-node/src/api/index.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as adapter from '@api3/airnode-adapter';
```

Comment:
Is the `api` folder the right place? Maybe better move this folder to handlers?

Reply:
I didn't really find the right place that's why I created an `api` directory. I was kinda following the pattern used in `initializeProvider` and `processTransactions` handlers - there's a "stub" code in `handlers` directory calling an actual code from `evm` directory. Since API calls are not related to EVM I created a separate directory.

Reply:
Let's keep it this way - we can think about moving this to some other place later.